File: calculator.java
package springIntro;

import java.util.*;

public class calculator {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		System.out.println("Which do you want make proccess");
		char operators = in.next().charAt(0);
		float number1, number2, result = 0;
		switch(operators) {
		case '+':
			System.out.println("Enter the numbers");
			number1 = in.nextFloat();
			number2 = in.nextFloat();
			result = number1 + number2;
			break;
		case '-':
			System.out.println("Number and number to subtract = ");
			number1 = in.nextFloat();
			number2 = in.nextFloat();
			result = number1 - number2;
			break;
		case '*':
			System.out.println("Enter the numbers");
			number1 = in.nextFloat();
			number2 = in.nextFloat();
			result = number1 * number2;
			break;
		case '/':
			System.out.println("Number and number to divide = ");
			number1 = in.nextFloat();
			number2 = in.nextFloat();
			result = number1 / number2;
			break;
		}
		System.out.println("Result = " + result);
	}

}

File: calculator.cpp
#include<iostream>
using namespace std;
int main(){
	char operators;
	cout << "Which do you want make proccess = ";
	cin >> operators;
	float number1, number2, result = 0;
	switch(operators){
		case '+':
			cout << "Enter the numbers = ";
			cin >> number1 >> number2;
			result = number1 + number2;
			break;
		case '-':
			cout << "Number and number to subtract = ";
			cin >> number1 >> number2;
			result = number1 - number2;
			break;
		case '*':
			cout << "Enter the numbers = ";
			cin >> number1 >> number2;
			result = number1 * number2;
			break;
		case '/':
			cout << "Number and number to divide = ";
			cin >> number1 >> number2;
			result = number1 / number2;
	}
	cout << "Result = " << result;
	return 0;
}
